### DISCUSSION TITLE
Windows Package Manager v1.0.12576

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-cli#1475] [General]
This release includes several new features related to the experimental Microsoft Store source. The REST API now has support for source level agreements, and an HTTP header pass through. Packages can also have agreements a user must accept before downloading and installing a 
package. We have also made improvements for handling silent installation with MSI UAC issues, and deferred registration for MSIX packages. A new experimental feature will show dependencies listed in a package manifest. The COM API is now considered a stable feature, and has been removed from experimental features. We have also started the work to begin supporting the new v1.1 schema #1243. The implementation for these new keys will follow in subsequent releases.

Thanks to @ChungZH for making some UX improvements to show how many upgrades are available, and displaying the version number during install.

## Features
#200 Require EULA/TOS acceptance before download starts.
#893 Add support for an arbitrary HTTP header value in REST API
#967 Max Installer nodes 1024 
#1012 *Experimental* Show dependencies
#1174 Added "doProgressTimeoutInSeconds" Setting 
#1216 Add Microsoft Store REST Source as default option and fix telemetry gaps
#1337 Show the version number during install.
#1354 winget upgrade: Display count of available upgrades
#1396 Add support for rest api 1.1 interface
#1397 Add deferred registration for MSIX
#1398 Use MSI API to allow UAC prompts on MSI silent installs
#1400 Client verbose logging does not log sufficient information to diagnose issues interacting with rest sources. 
#1419 Remove the packagedAPI experimental feature flag

## Bugs
#1406 InstallerSuccessCodes in manifest schema does not provide any numerical limits
#1416 winget source add doesn't warn you when adding an unsupported source.

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v1.0.12576'>Windows Package Manager v1.0.12576</a>.</em>

**Comment by jedieaston** [comment]
> Max Installer nodes 1024

Since this change was only made to schema 1.1, I'm assuming we can't submit manifests with, say, [500-ish installers](https://download.cdn.mozilla.net/pub/firefox/releases/92.0/) (97 languages * 5 installer types) quite yet, right?

**Reply by jedieaston** [reply]
Also, re: schema v1.1, can the aka.ms links for the schema files be created for 1.1 now that it's merged to master? They're cleaner than the raw.githubusercontent.com ones.

**Reply by denelon** [reply]
You are correct, and thank you for reminding me! It's been a whirlwind few weeks.

**Reply by denelon** [reply]
They should all be working now.

**Comment by jedieaston** [comment]
Also, this release doesn't seem to work on ARM64:

<img width="666" alt="image" src="https://user-images.githubusercontent.com/21368066/133806399-5d7718d4-b5a2-4748-b438-884f7807cd2f.png">

**Reply by denelon** [reply]
Thanks for pointing that out! We will take a look now.

**Reply by JohnMcPMS** [reply]
I was not able to repro on my device.

I think the error that you are seeing is due to a problem with your current install of the AppInstaller, not this package.  You should be able to use the settings app to reset the package or PowerShell to install this new one which should hopefully get everything right again.

**Reply by jedieaston** [reply]
It worked! Thanks!

**Comment by ItzLevvie** [comment]


**Reply by denelon** [reply]
Thanks for catching that. I updated the file name.
